app/src/main/java/org/walleth/functions/Formatting.kt

```diff
@@ -3,15 +3,23 @@ package org.walleth.functions
 import org.walleth.data.tokens.Token
 import java.math.BigDecimal
 import java.math.BigInteger
-import java.math.MathContext
-import java.math.RoundingMode
+import java.text.DecimalFormat
+
+val formatter: DecimalFormat
+    get() = DecimalFormat("#.#####")
```

Comment:
Does this have an advantage over just `val formatter =  DecimalFormat("#.#####")` ?